### ISSUE TITLE
Stripe Express accounts

### ISSUE THREAD

**Issue by Changaco** [liberapay/liberapay.com#2238]
(This issue replaces <https://github.com/liberapay/salon/issues/475>.)

Adding support for Stripe Express would: simplify the setup for creators who don't already have a standard Stripe account; provide an alternative to creators who don't want to connect their standard Stripe account to Liberapay (#1864); and possibly unblock users who have been rejected by Stripe (#1755).

The countries supported by Stripe Express appear to be the same ones as for Standard accounts, so no benefit there.

The disadvantages of supporting Express accounts are:

1. Liberapay could go bankrupt in case of fraud (“For Express and Custom accounts, the platform is ultimately responsible to cover negative balances”, [Understanding Connect account balances](https://stripe.com/docs/connect/account-balances#accounting-for-negative-balances)). This means that we would have to manually review and approve accounts, before allowing them to receive payments or before allowing them to withdraw their earnings. Ideally, Liberapay would move to a limited liability legal structure before taking on more risk, even if that risk is very small.
2. There would be extra fees on payouts (see [Connect Pricing](https://stripe.com/connect/pricing)), and it's not clear to me how Liberapay would recover them.
3. ~~Liberapay would once again be responsible for providing transaction records to users ([#714](https://github.com/liberapay/liberapay.com/issues/714)), because [the Express dashboard doesn't support exporting data](https://support.stripe.com/express/questions/can-i-export-a-list-of-my-payouts-from-stripe).~~

**Comment by Changaco** [comment]
Closing in favor of #2324, as I don't think Express accounts make much sense anymore now that there are [drop-in components for the creation and management of Custom accounts](https://docs.stripe.com/connect/supported-embedded-components).
